src/store/createAccessorStore.js

```diff
@@ -0,0 +1,43 @@
+import { hasSetter } from "@typhonjs-fvtt/runtime/svelte/util";
```

Comment:
Where does this come from? I can't find it [here](https://github.com/typhonjs-fvtt-lib/svelte/tree/main/src/util) and I'm not sure where else it might be

Reply:
There is a base generic svelte lib with nothing to do w/ Foundry. 

https://github.com/typhonjs-svelte/lib/tree/main/src

`@typhonjs-svelte/lib` is pulled into `@typhonjs-fvtt/svelte`; so all of @typhonjs-svelte/lib is also a part of `@typhonjs-fvtt/svelte`.  See:
https://github.com/typhonjs-fvtt-lib/svelte/blob/main/src/util/index.js#L1

I'm moving everything that doesn't have to do w/ Foundry to @typhonjs-svelte/lib eventually. 

TRL works on Foundry, but I'm angling it to be a library that work anywhere in time.

Reply:
@typhonjs-fvtt/runtime  seems like a mono-repo, but it's pulled together from a couple of parent repos. Indeed documentation and all that forthcoming.